copyrightStatement.txt

```diff
@@ -0,0 +1,12 @@
+<!--DISCLAIMER-->
+<html>
+<p>Copyright 2015 RTE (France), SmarTS Lab (Sweden), AIA (Spain) and DTU (Denmark)</p>
+<p>?- ?RTE: http://www.rte-france.com/</p>
```

Comment:
right... I got a prompt of character conversion... but didn't pay attention to it, cause these stuff usually happens on windows machines. 
Glad you found that error :-)